doc/tech/usecases.md

```diff
@@ -11,3 +11,16 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+
+2-Objective :
+
+
+Based on CNTT’s Project goals & Purpose was formulated is to develop a robust infrastructure model and define a limited discrete set of architectures built on that model that can be tested and validated for use across the entire member community. 
+
+CNTT Scope is extending beyond the Cloud infrastructure that should build into the Regional or National Data Center  but it extended to Edge , which CNTT consider it as a disruptive  approach and use case that CNTT will add value especially there is a lot of scattered project/Initiative either under Standard Development bodies or  Open source communities but there is no 
+
+So Object of this Edge is :
+-	To Harmonize the work under Standard Development bodies or  Open source communities 
+-	To build Common Cloud infrastructure based on CNTT principles that can be consumed by any operator
```

Comment:
Modify the existing RM and RA-s so that they are aligned with the edge requirements

Reply:
done